operators/kubecost-operator/ci.yaml

```diff
@@ -2,36 +2,37 @@
 cert_project_id: 667b9622f8a08cf49ff17c1f
 fbc:
   catalog_mapping:
-  - catalog_names:
-    - v4.12
-    template_name: v4.12.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.13
-    template_name: v4.13.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.14
-    template_name: v4.14.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.15
-    template_name: v4.15.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.16
-    template_name: v4.16.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.17
-    template_name: v4.17.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.18
-    template_name: v4.18.yaml
-    type: olm.template.basic
-  - catalog_names:
-    - v4.19
-    template_name: v4.19.yaml
-    type: olm.template.basic
+    - catalog_names:
+        - v4.12
+      template_name: v4.12.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.13
+      template_name: v4.13.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.14
+      template_name: v4.14.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.15
+      template_name: v4.15.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.16
+      template_name: v4.16.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.17
+      template_name: v4.17.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.18
+      template_name: v4.18.yaml
+      type: olm.template.basic
+    - catalog_names:
+        - v4.19
+        - v4.20
+      template_name: v4.19.yaml
+      type: olm.template.basic
```

Comment:
We should probably be creating a new entry in the `catalog_mapping` list here and templating the `v2.20.yaml`. I will address this in the next Kubecost release.